app/gui/src/project-view/components/GraphEditor/GraphNodeOutputPorts.vue

```diff
@@ -260,4 +270,28 @@ graph.value?.suggestEdgeFromOutput(outputHovered)
     calc(var(--node-size-y) + var(--output-port-max-width) + 16px)
   );
 }
+
+/**
+ * Extension of the output port's hover area, to ensure the button doesn't disappear as the mouse is
+ * moved toward it.
+ */
+.createNodeButtonApproachZone {
```

Comment:
Now, the button and its click zone are split into different modules, which makes changes to either one of them harder. But we can live with that.